cmd/prometheus/main.go

```diff
@@ -1089,10 +1089,10 @@ func main() {
 
 		callback := func(success bool) {
 			if success {
-				notifs.DeleteNotification(api.ConfigurationUnsuccessful)
+				notifs.DeleteNotification(api.ConfigurationUnsuccessfullText)
 				return
 			}
-			notifs.AddNotification(api.ConfigurationUnsuccessful)
+			notifs.AddNotification(api.ConfigurationUnsuccessfullText, api.ConfigurationUnsuccessfullDetails)
```

Comment:
Just a thought for later: Are you thinking of ever having varying details for the same text? If not, then bundling text+details into a struct and passing that in would reduce the degree of freedom that would allow using a text with the wrong details (especially if used from multiple places, which would also reduce repetition).

Reply:
Yep, sounds good for now.